### PR TITLE
adding the api to search for categories

### PR DESCRIPTION
**Pull request by twissy22**

### CODE REVIEW

**Review by Harrison365 — CHANGES_REQUESTED** [review]
Hi Adam :) 
Please could you add 404 error handling (including a test).
Then push and send us a new pull request. 
Otherwise, looks good 🥇

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+//             review_img_url: expect.any(String),
+//             created_at: expect.any(Date),
+//             votes: expect.any(Number),
+//             designer: expect.any(Number),
+//             comment_count: expect.any(Number)
+//           });
+//         });
+//       });
+//   });
+// });
```
Comment (emilyb93) [inline __tests__/app.test.js]: Remove any comments before requesting a PR, your pr should only contain code that is relevant to the ticket

File: app.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const app = express();
+
+const { getCategories, getReview } = require("./controllers/game.contoller.js")
+
+
+app.get("/api/categories", getCategories);
+// app.get("/api/reviews", getReview);
```
Comment (emilyb93) [inline app.js]: remove commented code

**Review by emilyb93 — APPROVED** [review]
Remove the commented code and then merge

**Pull request merged**